bin/kano-ui-autostart

```diff
@@ -75,6 +75,9 @@ systemctl --user start kano-home-button
 initflow=`sudo kano-init status`
 if [ "$initflow" != "disabled" ]; then
 
+    # Start the boards daemon so we can talk to hardware periherals during Overture
+    systemctl --user start kano-boards
```

Comment:
Just to make sure this is what you want here:

This will not start the `cpu-monitor` animation, for that you'd also need

```
systemctl --user start kano-speakerleds
```